graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/tweet/TopSecondDegreeByCountTweetRecsGenerator.java

```diff
@@ -26,128 +26,100 @@
 import com.twitter.graphjet.algorithms.RecommendationRequest;
 import com.twitter.graphjet.algorithms.RecommendationType;
 import com.twitter.graphjet.algorithms.TweetIDMask;
-import com.twitter.graphjet.algorithms.TweetRecommendationInfo;
+import com.twitter.graphjet.algorithms.counting.GeneratorHelper;
 import com.twitter.graphjet.hashing.SmallArrayBasedLongToDoubleMap;
-
-import it.unimi.dsi.fastutil.longs.LongArrayList;
-import it.unimi.dsi.fastutil.longs.LongList;
 import it.unimi.dsi.fastutil.longs.LongOpenHashSet;
 import it.unimi.dsi.fastutil.longs.LongSet;
 
 public final class TopSecondDegreeByCountTweetRecsGenerator {
-  private static final int MIN_USER_SOCIAL_PROOF_SIZE = 1;
-
-  private TopSecondDegreeByCountTweetRecsGenerator() {
```

Comment:
this private constructor and the final class ensure that it will not be instantiated or subclassed by anyone. could you add it back?